packages/grafana-runtime/src/components/NewDataSourcePicker.tsx

```diff
@@ -0,0 +1,57 @@
+import { DataSourceInstanceSettings } from '@grafana/data';
+import { DataSourceRef } from '@grafana/schema';
+
+interface NewDataSourcePickerProps {
+  onChange: (ds: DataSourceInstanceSettings) => void;
+  current: DataSourceRef | string | null; // uid
+  hideTextValue?: boolean;
+  onBlur?: () => void;
+  autoFocus?: boolean;
+  openMenuOnFocus?: boolean;
+  placeholder?: string;
+  tracing?: boolean;
+  mixed?: boolean;
+  dashboard?: boolean;
+  metrics?: boolean;
+  type?: string | string[];
+  annotations?: boolean;
+  variables?: boolean;
+  alerting?: boolean;
+  pluginId?: string;
+  /** If true,we show only DSs with logs; and if true, pluginId shouldnt be passed in */
+  logs?: boolean;
```

Comment:
```suggestion
  type?: string | string[];
```
do we need to expose these in runtime?  the usage/meaning of these varies so much that I am not really sure when|if they are useful

Reply:
I don't know 🤔 I pulled these from the interface that already exists.